gui/src/actions.js

```diff
@@ -2,23 +2,38 @@ import * as api from 'api'
 import { AuthenticationError } from 'errors'
 import { pascalCase } from 'change-case'
 
-const fetchActions = {}
-
-const createAction = (type) => ({type: type})
+const createAction = type => ({type: type})
 
-const requestNetworkError = (type, error) => ({
+const createErrorAction = (error, type) => ({
   type: type,
   error: error,
   networkError: true
```

Comment:
This is for another PR, but I think we should discuss or even kill ✂️ `networkError`.

Reply:
Agreed. It's lost it's original meaning.